static/js/pages/App.js

```diff
@@ -19,50 +20,93 @@ import AccountSettingsPage from "./settings/AccountSettingsPage"
 import users, { currentUserSelector } from "../lib/queries/users"
 import { routes } from "../lib/urls"
 
+import { ALERT_TYPE_TEXT } from "../constants"
 import type { Match } from "react-router"
 
+import { addUserNotification } from "../actions"
+
+import type { CurrentUser } from "../flow/authTypes"
+
 type Props = {
-  match: Match
+  currentUser: CurrentUser,
+  match: Match,
+  addUserNotification: Function
 }
 
-export default function App(props: Props) {
-  const { match } = props
+export class App extends Component<Props> {
+  componentDidMount() {
+    const { addUserNotification } = this.props
+    const notifications = document.querySelector(".site-wide")
+    if (notifications) {
+      const notificationId = notifications.getAttribute("data-notification-id")
+      const notificationMessage = notifications.getElementsByTagName("p")[0]
```

Comment:
Can we just use `notifications.innerText` or `notifications.textContent` here?

Reply:
that's is supposed to be the at the same line but linter is moving it to the next line.

Reply:
My build started to fail by moving it in the same line. It's asking to format it.

Reply:
Now going to revert it back.